settings.ts

```diff
@@ -0,0 +1,32 @@
+/** Constants */
+export const YOUTUBE_BASE_URL = "youtube.com";
+export const DEFAULT_TEMPLATE = "---\ndate: {{Date}}\n---\n# {{Title}}\n![]({{ImageURL}})\n## Description:\n{{Description}}\n-> [Youtube video Link]({{PodcastURL}})\n\n## Summary:\n";
+export const DEFAULT_DATE_FORMAT = "YYYY-MM-DD";
+export const DEFAULT_SUMMARY_SIZE = 3;
+export const MAX_SUMMARY_SIZE = 20;
+export const MAX_TOKEN_SIZE_IN_A_REQUEST = 2500;
+
+export interface PluginSettings {
+	/** Open API Key */
+	openAIApiKey: string;
+	
+	/** Minimum Summary Lines */
+	summarySize: number;
+
+    /** Max Token Size in a Request */
+    maxTokenSize: number;
```

Comment:
Can you explain why `maxTokenSize` is needed? The way I see that is that every model (gpt3.5, 4 etc.) has a different max token size and having lower token size in a request than the maximum would just increase the expensive requests to OpenAI, which would just cost more money without any benefits.

Reply:
Hi @ozdemir08 , understood.  I'm thinking to remove from the settings pane, but still keep it in `settings.ts`.